core/src/main/java/org/apache/calcite/plan/MaterializedViewSubstitutionVisitor.java

```diff
@@ -39,9 +39,9 @@
   private static final ImmutableList<UnifyRule> EXTENDED_RULES =
       ImmutableList.<UnifyRule>builder()
           .addAll(DEFAULT_RULES)
-          .add(ProjectToProjectUnifyRule1.INSTANCE)
-          .add(FilterToFilterUnifyRule1.INSTANCE)
-          .add(FilterToProjectUnifyRule1.INSTANCE)
+//          .add(ProjectToProjectUnifyRule1.INSTANCE)
+//          .add(FilterToFilterUnifyRule1.INSTANCE)
+//          .add(FilterToProjectUnifyRule1.INSTANCE)
```

Comment:
why remove these rules?

Reply:
I just want to illustrate that the new added rule set can cover all the existing tests.
Also I'm targeting to replace the existing rules with the new added rules.

Reply:
It works, so now you can remove them instead of commenting out.

Reply:
Thanks a lot @hsyuan 
In current, I'd prefer to commenting out the unused rules. Thus to keep this patch cleaner and easier for review.
If this patch is approved. I will remove the unused rules instantly.
Thanks again for looking into this and starting the review :)